src/autolinker.ts

```diff
@@ -107,20 +110,15 @@ function isCommonAbstractOp(op: string) {
 }
 
 function lookAheadBeyond(entry: BiblioEntry) {
-  if (entry.type === 'term') {
-    if (/^\w/.test(entry.key!)) {
-      return '\\b(?!\\.\\w|%%|\\]\\])';
-    } else {
-      return '';
-    }
-  } else {
-    // type is "op"
-    if (isCommonAbstractOp(entry.key!)) {
-      return '\\b(?=\\()';
-    } else {
-      return '\\b(?!\\.\\w|%%|\\]\\])';
-    }
+  if (entry.type === 'op' && isCommonAbstractOp(entry.key!)) {
+    // must be followed by parentheses
+    return '\\b(?=\\()';
+  }
+  if (entry.type !== 'term' || /^\w/.test(entry.key!)) {
+    // must not be followed by `.word` or `%%` or `]]`
+    return '\\b(?!\\.\\w|%%|\\]\\])';
```

Comment:
I went with comment re-phrasing what the regexp says, instead of putting it in an appropriately named variable. The regexp is pretty dense, so having it spelled out can be helpful.